qark/plugins/helpers.py

```diff
@@ -39,13 +43,17 @@ def get_min_sdk(manifest_xml):
         return 1
 
 
-def get_target_sdk(manifest_xml):
+def get_target_sdk(manifest_xml, files=None):
     """
     Given the manifest as a `minidom.parse`'d object, try to get the target SDK the manifest specifies.
 
     :param manifest_xml: object after parsing the XML
+    :param Set[str] files: list of files received from Scanner
     :return: int of the version if it exists, else 1 (the default)
     """
+    if files:
+        manifest_xml = get_manifest_out_of_files(files)
```

Comment:
Are we overwriting the manifest_xml here? is there a chance of us passing in the wrong one? If so, add details to the docstring

Reply:
This hits when `manifest=None` but can be improved by checking `manifest is None` I suppose.